crates/bevy_solari/src/pathtracer/pathtracer.wgsl

```diff
@@ -80,12 +92,13 @@ fn pathtrace(@builtin(global_invocation_id) global_id: vec3<u32>) {
 
 struct NextBounce {
     wi: vec3<f32>,
+    mis_weight: f32,
     pdf: f32,
 }
 
 fn importance_sample_next_bounce(wo: vec3<f32>, ray_hit: ResolvedRayHitFull, rng: ptr<function, u32>) -> NextBounce {
-    let diffuse_weight = 1.0 - ray_hit.material.metallic;
-    let specular_weight = ray_hit.material.metallic;
+    let diffuse_weight = mix(mix(0.4f, 0.9f, ray_hit.material.roughness), 0.f, ray_hit.material.metallic);
```

Comment:
Theoretically the best way to do this would be sampling a lobe based on fresnel, but I felt like keeping it simple. This is good too.

Did you want to use roughness, or linear roughness there though?

Reply:
It's not a very good way to importance sample, might as well try perceptual roughness if that is better.